static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
The prefix should not be part of the field value. To minimize the required
storage and communication, the field values in the DB should be integers.
The JavaScript code should translate them into strings, and add the prefix
according to the localization.
Each field that is present in the description should be imported. If it's
easier to just load all of them, then that would be best.

On Tue, May 12, 2015, 15:40 omerxx <notifications@github.com> wrote:

> In static/js/marker.js
> <https://github.com/hasadna/anyway/pull/250#discussion_r30128961>:
>
> > @@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
> >
> >  		this.$el.width(400);
> >  		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
> > -		this.$el.find(".description").text(this.model.get("description"));
> > +		this.$el.find(".roadType").text(this.model.get("roadType"));
>
>
>    1. What prefix is that? doesn't the strings present as they should?
>    2. Correct me if I'm wrong - process.py fills the DB. where should the
>    localization be done if so?
>    3. load_extra_data pulls many other properties, would you like each
>    option to have a separate field?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/250/files#r30128961>.
>